site/docs/numbers/porting/csrLookupTool.mdx

```diff
@@ -32,47 +32,18 @@ This walks through how to programmatically lookup Customer Service Records (CSRs
-
-<Highlight color="#079CEE">POST</Highlight> <code>https://dashboard.bandwidth.com/api/accounts/{accountId}/csrs</code>
-
-### Request Parameters
-
-| Parameter | Required | Description |
-|:--------- |:--------:|:------------|
-| CustomerOrderId | No | Internal customer order id for tracking the order. Only alphanumeric values, dashes and spaces are allowed. Max length is 40 characters. |
-| WorkingOrBillingTelephoneNumber (WTN) | Yes | Working or Billing telephone number in 10-digits format NPANXXXXXX. |
-| AccountNumber | No | Identifies the main account number on your bill, assigned by the current service provider. Alphanumeric characters are supported with a max length of 20 characters. |
-| AccountTelephoneNumber | No | Identifies the account telephone number assigned by the current service provider. Alphanumeric characters are supported with a max length of 10 characters. |
-| EndUserName | No | Identifies the name of the end user associated with the telephone number being queried. Alphanumeric characters are supported with a max length of 100 characters. |
-| AuthorizingUserName | No | Identifies the name of the end user who signed the authorization. Alphanumeric characters are supported with a max length of 100 characters. |
-| CustomerCode | No | Identifies the customer code associated with the service provider. Must be numeric characters only with a max length of 3 characters. |
-| EndUserPIN | No | Identifies the end user’s personal identification number (PIN). Alphanumeric characters are supported with a max length of 50 characters. |
-| EndUserPassword | No | Identifies the end user’s password to access the CSR, if applicable. Alphanumeric characters are supported with a max length of 100 characters. |
-| AddressLine1 | No | Identifies the address line 1 portion of the service address. Alphanumeric characters are supported with a max length of 200 characters. |
-| City | No | Identifies the city of the end user where the telephone number is being serviced. Alphanumeric characters are supported with a max length of 100 characters. |
-| State | No | Identifies the abbreviation for the state or province of the end user where the telephone number is being serviced. Must be alpha characters only in the format XX with a max length of 2 characters, where XX is the state or province abbreviation. |
-| ZIPCode | No | Identifies the ZIP code, ZIP Code + extension, or postal code or the end user where the telephone number is being serviced. Alphanumeric characters are supported with a max length of 10 characters (including the dash if using ZIP Code with extension). |
-| TypeOfService | No | The type of service identifying the end user account. Must be alphabetic characters only with a max length of 50 characters. |
+Before creating the order, be sure to collect any information from the end user to submit to the [/csrs API](/apis/numbers/#operation/CreateCsrOrder). Generally speaking, the more information provided, the better.
```

Comment:
Suggest changing this to "Before creating the order, collect any information from the end user as described in the request body schema for the POST method [/csrs API](/apis/numbers/#operation/CreateCsrOrder)."